lib/OpenQA/WebAPI.pm

```diff
@@ -296,19 +296,26 @@ sub startup {
     my $api_r_job = $api_job_auth->route('/')->to(namespace => 'OpenQA::WebAPI::Controller::API::V1');
     $api_r_job->get('/whoami')->name('apiv1_jobauth_whoami')->to('job#whoami');    # primarily for tests
 
+    # api/v1/job_groups
+    $api_public_r->get('/job_groups')->name('apiv1_list_job_groups')->to('job_group#list');
+    $api_public_r->get('/job_groups/:group_id')->name('apiv1_get_job_group')->to('job_group#list');
+    #$api_ra->post('/job_groups/:group_id')->name('apiv1_post_job_group')->to('job_group#create');
+    $api_ra->put('/job_groups/:group_id')->name('apiv1_put_job_group')->to('job_group#update');
+    #$api_ra->delete('/job_groups/:group_id')->name('apiv1_delete_job_group')->to('job_group#delete');
```

Comment:
I don't think those lines are problematic. It should be quite obvious what the lines are for and that they are disabled because the related back-end methods haven't been implemented yet.

And for the record, I think this pull request removes more out-commented code that it introduces.